calls/forms.py

```diff
@@ -1,13 +1,72 @@
+from bsd.models import BSDProfile
 from django import forms
 from django.conf import settings
+from django.contrib.auth.models import User
 from django.forms import widgets
-from .models import CallCampaign
+from django.forms.widgets import Widget
+from django.forms.utils import flatatt
+from django.utils.html import mark_safe
+from .models import CallCampaign, CallProfile
 
 CALLS_MAX_DISTANCE_MILES = settings.CALLS_MAX_DISTANCE_MILES
 CALLS_MAX_LIST_SIZE = settings.CALLS_MAX_LIST_SIZE
 
+class CommaSeparatedCallersTextArea(Widget):
+    def render(self, name, value, attrs=None, renderer=None):
+        final_attrs = self.build_attrs(attrs, {'type':'text', 'name':name})
```

Comment:
is `'name':name` correct? i dont really understand what this does but seems like an odd value to put for name of something.

Reply:
Can you clarify here? Name is an html attribute that django uses to populate data to correct field.